patroni/ha.py

```diff
@@ -1367,8 +1367,12 @@ def _run_cycle(self):
 
     def run_cycle(self):
         with self._async_executor:
-            info = self._run_cycle()
-            return (self.is_paused() and 'PAUSE: ' or '') + info
+            try:
+                info = self._run_cycle()
+                return (self.is_paused() and 'PAUSE: ' or '') + info
+            except Exception:
+                logger.exception('Unexpected exception')
+                return 'Unexpected exception raised, please report it as a BUG'
```

Comment:
We should tell what went wrong, no?
If a user someday files a bug report with `Unexpected exception`, would we immediately know where to look at? :smile:

Reply:
`logger.exception()` producing the stack trace